media/src/main/java/edu/artic/media/audio/AudioPlayerService.kt

```diff
@@ -144,16 +150,24 @@ class AudioPlayerService : Service() {
 
     val disposeBag = DisposeBag()
 
+    @Inject
+    lateinit var analyticsTracker: AnalyticsTracker
+
     override fun onCreate() {
+        AndroidInjection.inject(this)
```

Comment:
There is a `DaggerService` that does this for you, if you prefer.

Reply:
Updated.